Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.9.2-slim-buster as base
+RUN pip install poetry
+COPY poetry.lock pyproject.toml ./
+
+FROM base as production
+RUN poetry add gunicorn
+RUN poetry install
+COPY ./todo_app ./todo_app
+ENV WEBAPP_PORT=80
+EXPOSE ${WEBAPP_PORT}
+ENTRYPOINT ["poetry", "run", "gunicorn", "-b", "0.0.0.0:80", "todo_app.app:create_app()"]
```

Comment:
You could change this to `ENTRYPOINT poetry run gunicorn -b 0.0.0.0:$WEBAPP_PORT "todo_app.app:create_app()"` if you want to use the env var. Otherwise I'd suggest deleting `ENV WEBAPP_PORT=80` since it's basically unused.